### PR TITLE
Fixes #31

### LINKED ISSUE #31: The traceback is returned if the API key is not defined in Nautobot settings
### Environment
* Python version:  any
* Nautobot version:  any
* Nautobot Chatops Plugin version:  any
* nautobot-plugin-chatops-meraki version:  1.2.0

### Expected Behavior
The plugin should read the key either from settings or from an environmental variable as per documentation.

### Observed Behavior
If the key is not defined in the settings you get the following traceback:
```
nautobot_1  | Internal Server Error: /api/plugins/chatops/slack/slash_command/
nautobot_1  | Traceback (most recent call last):
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
nautobot_1  |     response = get_response(request)
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
nautobot_1  |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 70, in view
nautobot_1  |     return self.dispatch(request, *args, **kwargs)
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/django/utils/decorators.py", line 43, in _wrapper
nautobot_1  |     return bound_method(*args, **kwargs)
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
nautobot_1  |     return view_func(*args, **kwargs)
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 98, in dispatch
nautobot_1  |     return handler(request, *args, **kwargs)
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/nautobot_chatops/api/views/slack.py", line 104, in post
nautobot_1  |     registry = get_commands_registry()
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/nautobot_chatops/workers/__init__.py", line 70, in get_commands_registry
nautobot_1  |     command_func = worker.load()
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/pkg_resources/__init__.py", line 2450, in load
nautobot_1  |     return self.resolve()
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/pkg_resources/__init__.py", line 2456, in resolve
nautobot_1  |     module = __import__(self.module_name, fromlist=['__name__'], level=0)
nautobot_1  |   File "/usr/local/lib/python3.9/site-packages/nautobot_plugin_chatops_meraki/worker.py", line 26, in <module>
nautobot_1  |     MERAKI_DASHBOARD_API_KEY = settings.PLUGINS_CONFIG["nautobot_plugin_chatops_meraki"]["meraki_dashboard_api_key"]
nautobot_1  | KeyError: 'meraki_dashboard_api_key'
```

### Steps to Reproduce
1. Install the plugin version 1.2.0
2. Start Nautobot without the `nautobot-plugin-chatops-meraki` plugin settings in the configuration file.
3. Execute the `/meraki` command.

### PR DESCRIPTION
**Pull request by ubajze**
Fixes #31

### CODE REVIEW

**Comment by jeffkala** [comment]
@ubajze pylint is failing.

**Comment by ubajze** [comment]
@jeffkala I think it should be good now

File: nautobot_plugin_chatops_meraki/worker.py [hunk 1]
```diff
@@ -26,10 +26,10 @@
 
 try:
     MERAKI_DASHBOARD_API_KEY = settings.PLUGINS_CONFIG["nautobot_plugin_chatops_meraki"]["meraki_dashboard_api_key"]
-except KeyError:
+except KeyError as e:
```
Comment (jeffkala) [inline nautobot_plugin_chatops_meraki/worker.py]:
```suggestion
except KeyError as err:
```

File: nautobot_plugin_chatops_meraki/worker.py [hunk 2]
```diff
@@ -26,10 +26,10 @@
...
     MERAKI_DASHBOARD_API_KEY = os.getenv("MERAKI_DASHBOARD_API_KEY")
     if not MERAKI_DASHBOARD_API_KEY:
-        raise Exception("Unable to find the Meraki API key.")
+        raise Exception("Unable to find the Meraki API key.") from e
```
Comment (jeffkala) [inline nautobot_plugin_chatops_meraki/worker.py]:
```suggestion
        raise Exception("Unable to find the Meraki API key.") from err
```

**Review by jeffkala — CHANGES_REQUESTED** [review]

**Review by jeffkala — APPROVED** [review]

**Pull request merged**